src/components/DateRangePicker.jsx

```diff
@@ -79,6 +79,7 @@ const defaultProps = {
   isRTL: false,
   firstDayOfWeek: null,
   verticalHeight: null,
+  transitionDuration: 0.2,
```

Comment:
does this need a real defaultProp value? If this is set to `undefined`, then it will delegate to the `defaultProp` in the DayPickerRangeController instead, rather than duplicating it here.